src/Client/ClientBase.cpp

```diff
@@ -2497,6 +2549,19 @@ void ClientBase::init(int argc, char ** argv)
 
     readArguments(argc, argv, common_arguments, external_tables_arguments, hosts_and_ports_arguments);
 
+    /// Support for Unicode dashes
+    /// Interpret Unicode dashes as default hyphens
+    for (auto & arg : common_arguments)
+    {
+        // replace em-dash with double-hyphen
+        boost::replace_all(arg, "—", "--");
+        // replace en-dash with hyphen
+        boost::replace_all(arg, "–", "-");
```

Comment:
Ok. I think, it's better to treat all of them as `--`.

Reply:
fixed